docs/components/icons/index.md

```diff
@@ -8,9 +8,20 @@
 
 Warp's icon set is designed to help users understand actions, information and draw attention to elements.
 
-
 <components-status react='released' vue='released' elements='released' />
 
+## Examples
+
+<theme-switcher />
+
+<icon-example />
+
+## Usage
+
+<component-design-guidelines name="Warp - Components / Icons" link="" />
```

Comment:
missing link?

Reply:
Ah. That was one of the components that I wasn't sure where to link to in Figma. Obviously forgot to ask about this one. 😅 Any ideas?

Reply:
I could only find the one that @warp-ds/icons use to import icons, but that Figma file doesn't have any useful information atm. https://www.figma.com/file/yEx16ew6S0Xgd579dN4hsM/Warp---Icons?type=design&node-id=6011-1442&mode=design&t=NCcmBNZ6qFUuEXqy-0.

Reply:
I've added it... so I guess now it is up to our designers to fill the destination page with valuable info. 😄